README.md

```diff
@@ -145,6 +145,12 @@ Default: none
 
 Adds this path onto the less file paths in the source map.
 
+#### sourceMapFileInline
```

Comment:
I know @vladikoff merged.. but this needs to be changed in the build folder (or am I wrong @vladikoff ?)